tests/topotests/srv6_sid_manager/rt1/bgpd.conf

```diff
@@ -0,0 +1,67 @@
+frr defaults traditional
+!
+bgp send-extra-data zebra
+!
+hostname rt1
+password zebra
+!
+log stdout notifications
+log monitor notifications
+log commands
+!
+!debug bgp neighbor-events
+!debug bgp zebra
+!debug bgp vnc verbose
+!debug bgp update-groups
+!debug bgp updates in
+!debug bgp updates out
+!debug bgp vpn label
+!debug bgp vpn leak-from-vrf
+!debug bgp vpn leak-to-vrf
+!debug bgp vpn rmap-event
+!
+router bgp 1
+ bgp router-id 1.1.1.1
+ no bgp ebgp-requires-policy
+ no bgp default ipv4-unicast
+ neighbor fc00:0:6::1 remote-as 6
+ neighbor fc00:0:6::1 timers 3 10
+ neighbor fc00:0:6::1 timers connect 1
+ neighbor fc00:0:6::1 ttl-security hops 20
+ !
+ address-family ipv6 vpn
+  neighbor fc00:0:6::1 activate
+ exit-address-family
+ !
+ segment-routing srv6
+  locator loc1
+ !
+!
+router bgp 1 vrf vrf10
+ bgp router-id 1.1.1.1
+ no bgp ebgp-requires-policy
+ no bgp default ipv4-unicast
+ !
+ address-family ipv6 unicast
+  sid vpn export 65024
```

Comment:
Minor comment: Why this func value? Is there is anything blocking us to use 65324? The value 65024 might be "out-of-range" for default value range of func (starting from "0xff00" if I remember correctly) particular format uncompressed-f4024? It might be a good thing to avoid confusion in case when user switches the uncompressed-f4024 and consequently there is a risk that related func are failed to allocate?

Reply:
Tu further evolve this point: user should be notified directly if (s)he types the value of out-of-range func.